src/main/java/org/testng/internal/Invoker.java

```diff
@@ -1609,6 +1609,21 @@ public boolean isTrue(ITestNGMethod m, IClass c) {
     return result;
   }
 
+  private boolean isConfiguration(ITestNGMethod method) {
```

Comment:
Should be static, and should be moved into `BaseTestMethod` or in an `TestNgMethodUtils`-like class